src/App.tsx

```diff
@@ -21,14 +23,15 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
+              {isLoading && <Loader />}
+
               <TodoList />
```

Comment:
Better to show loader or list 
```suggestion
              {isLoading ? <Loader /> : <TodoList />}

```